salt/modules/win_lgpo.py

```diff
@@ -5738,6 +5738,7 @@ def _checkListItem(
     for list_element in xpath_object(policy_element):
         configured_items = 0
         required_items = 0
+
```

Comment:
You've changed nothing in this file. Could you please revert these empty lines to not include it to the PR?